src/scenes/AcceleratorRouter/Cohorts/CohortsListView.tsx

```diff
@@ -60,8 +62,19 @@ const CohortsListView = ({ filterModifiers, extraTableFilters }: CohortsListView
 
   const [hasFilters, setHasFilters] = useState<boolean>(false);
   const cohorts = useAppSelector(selectCohorts);
+  const [cohortsWithParticipantsNum, setCohortsWithParticipantNum] = useState<CohortWithParticipantNum[]>();
   const isEmptyState = useMemo<boolean>(() => cohorts?.length === 0 && !hasFilters, [cohorts?.length, hasFilters]);
 
+  useEffect(() => {
+    if (cohorts?.length) {
+      const newCohorts: CohortWithParticipantNum[] = cohorts.map((c) => ({
+        ...c,
+        numOfParticipants: c.participantIds?.length || 0,
+      }));
+      setCohortsWithParticipantNum(newCohorts);
+    }
+  }, [cohorts]);
```

Comment:
Instead of creating more state, you can determine this within the cell renderer. Inside the `CohortCellRenderer`, do something like this:

```
if (column.key === 'numOfParticipants') {
    return <CellRenderer {...props} value={((row as Cohort)?.participantIds || []).length} />;
  }
```

Reply:
Yes, this was done like this before, but the sorting uses the original value, not the rendered one, so we need the rows to have the quantity number value

Reply:
Oh I see. In that case maybe it should be hydrated into the data at the service level?